modules/cluster-logging-rn-5.6.adoc

```diff
@@ -0,0 +1,57 @@
+//included in cluster-logging-release-notes.adoc
+:_content-type: ASSEMBLY
+[id="cluster-logging-release-notes-5-6_{context}"]
+= Logging 5.6
+This release includes link:https://access.redhat.com/errata/[OpenShift Logging Release 5.6].
+//pending Errata creation
+
+[id="openshift-logging-5-6-enhancements_{context}"]
+== Enhancements
+//* (link:https://issues.redhat.com/browse/LOG-895[LOG-895])
+
+//* (link:https://issues.redhat.com/browse/LOG-2695[LOG-2695])
+
+//* (link:https://issues.redhat.com/browse/LOG-2715[LOG-2715])
+
+//* (link:https://issues.redhat.com/browse/LOG-2913[LOG-2913])
+
+//* (link:https://issues.redhat.com/browse/LOG-2222[LOG-2222])
+
+[id="openshift-logging-5-6-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, if you deleted the Kibana Custom Resource, the {product-title} web console continued displaying a link to Kibana. With this update, removing the Kibana Custom Resource also removes that link. (link:https://issues.redhat.com/browse/LOG-2993[LOG-2993])
+
+* Before this update, a user was not able to view the application logs of namespaces they have access to. With this update, the Loki Operator automatically creates a cluster role and cluster role binding allowing users to read application logs. (link:https://issues.redhat.com/browse/LOG-3072[LOG-3072])
```

Comment:
In your previous bullet you use second person `if you deleted the Kibana Custom Resource` 

Here, you use `user`: `a user was not able to view the application` 

Be consistent with which tense you use. Since IBM SG encourages us to use second person, that's what I'd go with. 

Nevertheless, both versions (you/user) are correct. So, pick whichever version requires less time and editing on your end.

Reply:
> Here, you use user: a user was not able to view the application -- here a "user" means not an "administrator" - you wouldn't be appropriate.